first-analysis-steps/prerequisites.md

```diff
@@ -85,7 +89,8 @@ If you can successfully execute all of the above steps, you are ready to go for
 the workshop!
 
 {% callout "Using Bash" %}
-The [Bash shell](http://en.wikipedia.org/wiki/Bash_(Unix_shell%29) will be used
+
+The [Bash shell](https://en.wikipedia.org/wiki/Bash_(Unix_shell%29) will be used
```

Comment:
This link seems to be broken

Reply:
Well spotted!

As it's an unpleasant URL for markdown parsing I've replaced it with a shortened version:  http://cern.ch/go/gdJ9